### ISSUE TITLE
[bug]: `ver'nightly'` fails to select nightly release of program

### ISSUE THREAD

**Issue by redxtech** [zdharma-continuum/zinit#483]
### What happened?

When using the `ver` ice on a program,  I wanted to select the latest nightly release. However, it is not working.

Leaving it omitted/empty defaults to `latest`, which is the latest stable release.

When I set it to `nightly`, it tells me `gh-r: failed to find the correct GitHub release asset to download.`

### Steps to reproduce

in my zshrc:

```zsh
zinit wait lucid from"gh-r" as"command" for \
    sbin"**/bin/nvim" ver"nightly"
```

when sourcing my zshrc, the error happens.

### Relevant output

```shell
gh-r: failed to find the correct GitHub release asset to download.
```

### Operating System & Version

OS: linux-gnu | Vendor: pc | Machine: x86_64 | CPU: x86_64 | Processor: unknown | Hardware: x86_64

### Zsh version

zsh 5.9 (x86_64-pc-linux-gnu)

### Terminal emulator

kitty, `xterm-kitty`

### If using WSL on Windows, which version of WSL

None

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by vladdoster** [comment]
@redxtech,

What is the output of:

```zsh
zinit version
```

**Comment by vladdoster** [comment]
@redxtech,

I can't reproduce this error

<img width="2056" alt="Screenshot 2023-03-10 at 01 15 15" src="https://user-images.githubusercontent.com/10052309/224248868-50dcae1e-6b0b-403b-9b78-eb5a5330fbf2.png">

**Comment by vladdoster** [comment]
@redxtech,

Indeed, the `version` subcommand is a recent addition to display the release tag + commits since.

Regarding Zinit being out-of-date...

I'd nudge you towards the `self-update` sub-command which will only attempt to update Zinit. While `update` will (afaik) accomplish what you want, it will attempt to update everything (not a great default 🤷‍♂️) which takes longer and might make an unwanted change or version update.
